tutorials/accelerated-python/notebooks/fundamentals/06__asynchrony__power_iteration.ipynb

```diff
@@ -1,397 +1,438 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "pszz-k8cDfqy"
+      },
+      "outputs": [],
+      "source": [
+        "!python power_iteration__async.py"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "VFCYIqwaKYqy"
+      },
+      "source": [
+        "## 7. Performance Analysis\n",
+        "\n",
+        "Before we profile the improved code, let's compare the execution times of both."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "uSPFNIb9KcPb"
+      },
+      "outputs": [],
+      "source": [
+        "power_iteration_baseline_output   = !python power_iteration__baseline.py\n",
+        "power_iteration_baseline_duration = float(power_iteration_baseline_output[-1].split()[-2])\n",
+        "power_iteration_async_output      = !python power_iteration__async.py\n",
+        "power_iteration_async_duration    = float(power_iteration_async_output[-1].split()[-2])\n",
+        "speedup = power_iteration_baseline_duration / power_iteration_async_duration\n",
+        "\n",
+        "print(f\"GPU Execution Time\")\n",
+        "print()\n",
+        "print(f\"power_iteration_baseline: {power_iteration_baseline_duration:.3f} s\")\n",
+        "print(f\"power_iteration_async:    {power_iteration_async_duration:.3f} s\")\n",
+        "print(f\"power_iteration_async speedup over power_iteration_baseline: {speedup:.2f}\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "o4WJVFBkkRaN"
+      },
+      "source": [
+        "Next, let's capture a profile report of our improved code."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "BtQR4CHikWFK"
+      },
+      "outputs": [],
+      "source": [
+        "!nsys profile --cuda-event-trace=false --capture-range=cudaProfilerApi --capture-range-end=stop --force-overwrite true -o power_iteration__async python power_iteration__async.py"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Pnvne_F4jYTh"
+      },
+      "source": [
+        "Finally, let's look at the profile in Perfetto and confirm we've gotten rid of the idling."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "mWXBvi-hFGhU"
+      },
+      "outputs": [],
+      "source": [
+        "!nsys export --type sqlite --quiet true --force-overwrite true power_iteration__async.nsys-rep\n",
+        "nsightful.display_nsys_sqlite_file_in_notebook(\"power_iteration__async.sqlite\", title=\"Power Iteration - Async Event\")"
+      ]
+    }
+  ],
+  "metadata": {
+    "accelerator": "GPU",
+    "colab": {
+      "gpuType": "T4",
+      "provenance": []
+    },
+    "kernelspec": {
+      "display_name": "Python 3 (RAPIDS 25.10)",
```

Comment:
This is the wrong kernelspec for this notebook. It should be "display_name": "Python 3 (ipykernel)". We don't have a RAPIDS 25.10 kernel in the Docker image. Please make sure this is fixed for all of the notebooks.